src/System.IO/src/System.IO.builds

```diff
@@ -25,6 +25,12 @@
     <Project Include="redist\System.IO.depproj">
       <TargetGroup>netcore50aot</TargetGroup>
     </Project>
+    <Project Include="System.IO.csproj">
+      <TargetGroup>uap101aot</TargetGroup>
+    </Project>
+    <Project Include="System.IO.csproj">
```

Comment:
I'm no sure where the binary build is going to be used (what's going to consume src/System.IO.builds). If you're saying that it's not necessary to invoke the csproj with TargetGroup=uap101, I can certainly remove it from System.IO.builds.

Updating the System.IO.Builds will not have an affect on runtime.any.system.io package. Could you please confirm that runtime.any.system.io is as you'd expect